cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaClusterRebalanceAssemblyOperator.java

```diff
@@ -282,6 +282,7 @@ public void onClose(KubernetesClientException e) {
 
     private Future<Void> reconcile(Reconciliation reconciliation, String host, CruiseControlApi apiClient, KafkaClusterRebalance clusterRebalance, State currentState, RebalanceAnnotation rebalanceAnnotation) {
         RebalanceOptions.RebalanceOptionsBuilder rebalanceOptionsBuilder = new RebalanceOptions.RebalanceOptionsBuilder();
+        // TODO: We also need to check if the user has asked for a verbose output
```

Comment:
so we haven't opened the PR against master yet so we could also think to remove the `verbose` property from the CRD but leaving the implementation in the client API. If we leave the verbose in the CRD we have to implement it because we are exposing it to the user while removing it, we can support it later.Wdyt?

Reply:
I think that makes sense. Lets remove the verbose option from the CRD and add a task to explore how best to show the verbose output to the user.

Reply:
Should I make the changes to the CRD?

Reply:
@tomncooper fine with me!